templates/conventional.hbs

```diff
@@ -0,0 +1,134 @@
+## Changelog
+
+{{!--
+Introduction
+• This template tries to follow conventional commits format https://www.conventionalcommits.org/en/v1.0.0/
+• The template uses regex to filter commit types into their own headings (this is more than just fixes and features headings)
+• It also uses the replaceText function in package.json to remove the commit type text from the message, because the headers are shown instead.
+
+• The text 'Breaking:' or 'Breaking changes:' can be located anywhere in the commit.
+• The types feat:, fix:, chore:, docs:, refactor:, test:, style:, perf: must be at the beginning of the commit subject with an : on end.
+   • They can optionally have a scope set to outline the module or component that is affected eg feat(bldAssess):
+• There is a short hash on the end of every commit that is currently commented out so that change log did not grow too long (due to some system's file size limitations).  You can uncomment if you wish [`{{shorthash}}`]({{href}})
+
+Example Definitions
+• feat: A new feature
+• fix: A bug fix
+• perf: A code change that improves performance
+• refactor: A code change that neither fixes a bug nor adds a feature
+• style: Changes that do not affect the meaning of the code (white-space, formatting, spelling mistakes, missing semi-colons, etc)
+• test: Adding missing tests or correcting existing tests
+• docs: Adding/updating documentation
+• chore: Something like updating a library version, or moving files to be in a better location and updating all file refs
+--}}
+
+
+{{!-- In package.json need to add this to remove label text from the change log output (becuase the markdown headers are now used to group them).
```

Comment:
```suggestion
{{!-- In package.json need to add this to remove label text from the change log output (because the markdown headers are now used to group them).
```